tools/font-loader-lib/index.js

```diff
@@ -0,0 +1,34 @@
+const fetchFonts = (window, document) => {
+    const head = document.querySelector('head');
+    const useFont = font => {
+        if (font.css) {
+            const style = document.createElement('style');
+            style.innerHTML = font.css;
+            head.appendChild(style);
+        }
+    };
+
+    const loadFonts = () => {
+        const iframe = document.createElement('iframe');
+        iframe.src = 'https://theguardian.com/font-loader';
+        iframe.classList = 'guardianFontLoader';
```

Comment:
The class `guardianFontLoader` is unnecessary, I'd only included as I was using it in my demo to apply a `display: none` style from my CSS.